jekyll/_cci2/workflows.md

```diff
@@ -430,6 +430,48 @@ workflows:
               ignore: /.*/
 ```
 
+In the example below, two jobs are defined (`test` and `deploy`) and three workflows utilize those jobs:
+
+- The `build` workflow runs for all branches except `main` and is not run on tags.
+- The `staging` workflow will only run on the `main` branch and is not run on tags.
+- The `production` workflow runs for no branches and only for tags starting with `v.`.
+
+```yaml
+workflows:
+  build: # This workflow will run on all branches except 'main' and will not run on tags
+    jobs:
+      - test:
+          filters:
+            branches:
+              ignore: main
+  staging: # This workflow will only run on 'main' and will not run on tags
+    jobs:
+      - test:
+          filters: &filters-staging
+            branches:
+              only: main
+            tags:
+              ignore: /.*/
+      - deploy:
+          requires:
+            - build
+          filters:
+            <<: *filters-staging
```

Comment:
Can we just add comments in for these lines to say we've using the same 'staging' filters for this job (same for use of alias in production), just to spell it out?

Reply:
I updated this with commit 6502576 can you check to see if that works?

Reply:
Great, thank you!!